lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -107,6 +122,16 @@
     key: privcertkey
     cert_chain: myverytrustedchain
 
+# Server certificate upload using key from ansible-vault
+tasks:
```

Comment:
It's not clearly stated anywhere in the developing modules pages so plenty of modules do it the wrong way. But that doesn't Mr we shouldn't do it the right way when we can 😊